pages/common/php-artisan.md

```diff
@@ -0,0 +1,19 @@
+# php artisan
+
+> Laravel's Artisan command line interface.
+
+- Display a list of all available commands:
+
+`php artisan help`
+
+- Start PHP's built-in web server for the current Laravel application:
+
+`php artisan serve`
+
+- Start an interactive PHP command line interface:
+
+`php artisan tinker`
+
+- Generate a new Eloquent model class with a migration, factory and resource controller:
```

Comment:
Is `Eloquent` meant to be capitalised?

Reply:
Yes, [Eloquent](https://laravel.com/docs/master/eloquent) is a noun - it is the name of a module in the Laravel framework.

Reply:
Cool, thanks :-)